app/org/sagebionetworks/bridge/validators/ActivityValidator.java

```diff
@@ -53,9 +53,6 @@ private void validate(Errors errors, TaskReference ref) {
     
     private void validate(Errors errors, SurveyReference ref) {
         errors.pushNestedPath("survey");
-        if (isBlank(ref.getIdentifier())) {
-            errors.rejectValue("identifier", CANNOT_BE_BLANK);
-        }
```

Comment:
Yes. They didn't have fail() in them and so it was succeeding even though not throwing exception. All are fixed.